src/main/java/it/eng/idsa/businesslogic/routes/CamelRouteReceiver.java

```diff
@@ -134,6 +157,107 @@ public void configure() throws Exception {
 				.process(sendDataToBusinessLogicProcessor);
 			//@formatter:on
 		}
+		
+		if (isEnabledIdscp2 && receiver) {
+			logger.info("Starting IDSCP v2 Server route");
+
+			from("idscp2server://0.0.0.0:29292?sslContextParameters=#sslContext&useIdsMessages=true")
+				.process(IdsMessageTypeExtractionProcessor)
```

Comment:
Currently it seems meaningless, but when we have to manage also contract negotiation it will be useful, becuse we need to manage all the message types on the same route